code/datums/mood_events/generic_negative_events.dm

```diff
@@ -201,6 +201,22 @@
 	var/unhinged = uppertext(unstable.Join(""))//example Tinea Luxor > TINEA LUXORRRR (with randomness in how long that slur is)
 	description = "THEY NEEEEEEED [unhinged]!!"
 
+/datum/mood_event/paranoid_close
+	description = "This person is too close to me, he has something in his mind?"
```

Comment:
```suggestion
	description = "Why do they all have to stand so close to me? What are they planning?"
```
Since we're not specifying individuals we should be using phrasing that acknowledges the potential for multiple people being nearby.